src/features/authentication/controller/auth.controller.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller, Post, Body, UsePipes, ValidationPipe } from "@nestjs/common";
+import { AuthService } from "@/features/authentication/service/auth.service";
+import { UserLoginDto } from "@/features/authentication/dto/userLogin.dto";
+import { UserDto } from "@/features/authentication/dto/user.dto";
+
+@Controller('api')
```

Comment:
```suggestion
@Controller('api/users')
```

có nên move phần `/users` lên đây cho gọn ko em nhỉ?

Reply:
à, 2 chức năng getUser với updateUser sẽ viết trong User controller em nhé, ko đặt trong Auth controller nữa.

auth controller chỉ nên dùng cho bộ 3 signup/login/logout thôi